zebedee-cms/src/main/java/com/github/onsdigital/zebedee/filters/AuthenticationFilter.java

```diff
@@ -27,6 +27,8 @@
 			.add(ReIndex.class)
 			.add(Ping.class)
 			.add(ClickEventLog.class)
+			.add(Identity.class)
+			.add(Service.class)
```

Comment:
I think these should be authenticated endpoints.

Reply:
The service endpoint shouldn't be there, but the identity endpoint should still work without the florence token